tests/test_micropipenv.py

```diff
@@ -104,6 +104,22 @@ def check_generated_pipfile_lock(pipfile_lock_path, pipfile_lock_path_expected):
     os.remove(pipfile_lock_path)
 
 
+@pytest.mark.online
```

Comment:
Thanks for the test! Two things we can improve:
* This test does not need internet so you can remove the `mark.online` decorator.
* The code in the test is repeating so it would be better to simplify it and use pytest parametrization. Take a look at other tests where we use it heavily.